### DISCUSSION TITLE
希望能在windows上面进行安装使用

### DISCUSSION THREAD

**Discussion by setruth** [linuxdeepin/developer-center#5563] [Features Request & Ideas | 特性请求 & 头脑风暴]
希望能尽快支持多平台在windows上面进行安装使用，并且减轻一下安装的方式，不是手动进行编译安装

**Comment by BLumia** [comment]
你反馈指的是 deepin-unioncode 这个项目吗？如果是的话，参见 https://github.com/orgs/linuxdeepin/discussions/5430

**Reply by setruth** [reply]
好的谢谢
